dird: director tries to connect to client when connection is disabled (#1099)
bareos/bareos: CHANGELOG.md

- Fix for wrong update message when updating all volumes from all pools with no existing volumes [PR #1015]
- Fix context confusion in Director's Python plugins [PR #1047]
- Fix several cases of undefined behaviour, memory corruption and memory leaks [PR #1060]

> - Fix case when director tries to connect to a client when it is not allowed to do so. [PR #1099]

Reviewer: ```suggestion
- Fix director connects to client while `Connection From Director To Client` is disabled. [PR #1099]
```
